### PR TITLE
Support BoringSSL in Envoy

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Release Notes.
 #### Features
 * Support `NETWORK` Profiling.
 * Let the logger as a configurable module.
-* Support analyze the data of OpenSSL library in `NETWORK` Profiling.
+* Support analyze the data of OpenSSL, BoringSSL library in `NETWORK` Profiling.
```
Comment (wu-sheng) [inline CHANGES.md:10]: Do we have a doc about network profiling in the rover or backend? If so, we should update the doc about this too.

**Review by wu-sheng — APPROVED** [review]
Generally good. We should consider e2e for this later.

File: CHANGES.md (hunk 1 above)
Comment (mrproliu) [reply CHANGES.md:10]: No, we haven't the doc to explain this. Update the document to explain which type of profiling the rover is supported?

Comment (wu-sheng) [reply CHANGES.md:10]: Explain which protocols/deployments we could support. I think we have ON and OFF CPU profiling in rover, right?

Comment (mrproliu) [reply CHANGES.md:10]: I have updated the configuration file to explain the ON/OFF CPU and Network profiling task.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**